tests/test_verbs.py

```diff
@@ -119,7 +143,37 @@ def test_identify(app):
         assert len(granularity) == 1
         description = tree.xpath('/x:OAI-PMH/x:Identify/x:description',
                                  namespaces=NAMESPACES)
-        assert len(description) == 2
+
+        friends_element = description[0]
+        for element in friends_element.getchildren():
+            for child in element.getchildren():
+                assert child.tag == \
+                    '{http://www.openarchives.org/OAI/2.0/friends/}baseURL'
+
+        eprints_root = description[1]
+        children = eprints_root[0].getchildren()
+        assert children[0].tag == \
```

Comment:
The order of the eprints children should not matter, hence a test that asserts that ``child[0] == content``, ``child[1]==metadataPolicy``, ``child[2]==dataPolicy``, is not testing anything useful, instead, make assertions on the lenght of children (4) and make sure that each of those is present in the children of eprints. Furthermore check for the contents of the eprints children, like:
```
content_children = children[0].getchildren()
assert len(content_children) == 1
assert content_children[0].tag == '{http://www.openarchives.org/OAI/2.0/eprints}URL'
assert content_children[0].text == 'http://arXiv.org/arXiv_content.htm'
(...)
metaPoli_children = children[1].getchildren()
assert len(metaPoli_children) == 2
...
```